tfhe/c_api_tests/test_compressed_list.c

```diff
@@ -0,0 +1,145 @@
+#include "tfhe.h"
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+int main(void) {
+  int ok = 0;
+
+  // First, we create a ClientKey and a CompactPublicKey
+  ClientKey *client_key = NULL;
+  ServerKey *server_key = NULL;
+
+  {
+    ConfigBuilder *builder;
+    Config *config;
+    ok = config_builder_default(&builder);
+    assert(ok == 0);
+
+    config_builder_enable_glwe_packing_compression(&builder);
```

Comment:
..._enable_compression or ..._enable_post_computation_compression

this does not take parameters as input ?

Reply:
Renamed `config_builder_enable_compression`

> this does not take parameters as input ?

No yet, see https://github.com/zama-ai/tfhe-rs/pull/1202#discussion_r1643015996

Reply:
> this does not take parameters as input ?

Done